shared/src/main/java/com/couchbase/lite/ReadOnlyDictionary.java

```diff
@@ -99,7 +104,7 @@ public Number getNumber(String key) {
     @Override
     public int getInt(String key) {
         FLValue value = fleeceValue(key);
-        return value != null ? value.asInt() : 0;
+        return value != null ? (int)value.asInt() : 0;
```

Comment:
Fleece has one type for integral numbers. It is Integer. 
Fleece's Integer is equivalent with Java Long. 2^64.